src/auth/tenant.ts

```diff
@@ -123,6 +130,13 @@ export class Tenant {
   private readonly emailSignInConfig_?: EmailSignInConfig;
   private readonly multiFactorConfig_?: MultiFactorAuthConfig;
 
+  /**
+   * The SMS Regions Config to update a tenant.
+   * Configures the regions where users are allowed to send verification SMS.
+   * This is based on the calling code of the destination phone number.
```

Comment:
country code?

Reply:
communicated offline, country code is not a standard google term in this case.